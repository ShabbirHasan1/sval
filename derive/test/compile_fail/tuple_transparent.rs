use sval_derive::*;

#[derive(Value)]
#[sval(transparent)]
pub struct Tuple(i32, i32, i32);

fn main() {

}