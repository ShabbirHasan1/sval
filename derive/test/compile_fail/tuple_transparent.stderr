error: proc-macro derive panicked
 --> compile_fail/tuple_transparent.rs:3:10
  |
3 | #[derive(Value)]
  |          ^^^^^
  |
  = help: message: unsupported attribute `transparent` on struct
